test_scripts/Capabilities/PersistingHMICapabilities/common.lua

```diff
@@ -118,20 +118,29 @@ function m.checkContentOfCapabilityCacheFile(pExpHmiCapabilities)
       for req, params in pairs(requests) do
         for _, param in ipairs(params) do
           local message = mod .. "." .. param
-          if param == "audioPassThruCapabilitiesList" then
-            if not utils.isTableEqual(cacheTable[mod].audioPassThruCapabilities, expHmiCapabilities[mod][req].params[param]) then
-              errorMessages = errorMessages ..
-                errorMessage(message, cacheTable[mod].audioPassThruCapabilities,
-                  expHmiCapabilities[mod][req].params[param])
+          local expectedResult = expHmiCapabilities[mod][req].params[param]
```

Comment:
@VjKlepikov Please read capabilities from the same file which is checked in previous line.
```
 86: if SDL.AppStorage.isFileExist(actions.sdl.getSDLIniParameter("HMICapabilitiesCacheFile")) then
 87:  local  cacheTable = utils.jsonFileToTable(hmiCapCacheFile)
```